pkg/neonvm/ipam/ipam.go

```diff
@@ -186,6 +196,20 @@ func LoadFromNad(nadConfig string, nadNamespace string) (*IPAMConfig, error) {
 	return n.IPAM, nil
 }
 
+func (i *IPAM) runIPAMWithMetrics(ctx context.Context, action ipamAction, actionName string) (net.IPNet, error) {
+	timer := i.metrics.StartTimer(actionName)
+	// This is if we get a panic
+	defer timer.Finish(IPAMUnknown)
```

Comment:
question: If this is only when we get a panic, should we indicate it as such?

Reply:
Done.